modules/nf-core/ampcombi/main.nf

```diff
@@ -0,0 +1,45 @@
+process AMPCOMBI {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::ampcombi=0.1.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:0.1.3--pyhdfd78af_0':
+        'quay.io/biocontainers/ampcombi:0.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta),  path(amp_input)
+    path(faa_folder)
+    val(outdir)
```

Comment:
Why a dedicted input channel and not with not `$prefix`?

Reply:
We thought this way we relate the sample_name to the respective AMP-tool-outputs. Also, AMPcombi may have two kinds of input: the path to the folder which then contains all AMP-tools' outputs to be summarised or a list of file-paths to the output-files to be summarised. The second option still needs a test-workflow, but we need to fix a bug in AMPcombi first.

Reply:
Mmm ok... I don't think I'm following... Maybe we need a real time meeting and do live demos 😅

Reply:
Agreed!